book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
```

Comment:
- [ ] Kindly make sure to add a method for adding rentals in the `Book` class
e.g
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
 end
```